packages/textmate-language/textmate-language.0.3.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Tokenizing code with TextMate grammars for syntax highlighting"
+description: """
+
+This package provides functions for reading TextMate grammars and tokenizing
+code on a line-by-line basis. `textmate-language` can read grammars from the
+document types of the `plist-xml`, `ezjsonm`, and `yojson` libraries."""
+maintainer: ["Alan Hu <alanh@ccs.neu.edu>"]
+authors: ["Alan Hu <alanh@ccs.neu.edu>"]
+license: "MIT"
+homepage: "https://github.com/alan-j-hu/ocaml-textmate-language"
+doc: "https://alan-j-hu.github.io/ocaml-textmate-language/"
+bug-reports: "https://github.com/alan-j-hu/ocaml-textmate-language/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "oniguruma" {< "0.2.0"}
+  "plist-xml" {with-test}
+  "alcotest" {>= "1.4" & < "2" & with-test}
+  "ezjsonm" {>= "1.2" & < "2" & with-test}
+  "yojson" {>= "1.7" & < "2" & with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/alan-j-hu/ocaml-textmate-language.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/alan-j-hu/ocaml-textmate-language.git"
x-ci-accept-failures: [
  "centos-7"
  "oraclelinux-7"
  "oraclelinux-8"
]
```

Reply:
Is it better to add those restrictions to https://github.com/ocaml/opam-repository/blob/master/packages/conf-oniguruma/conf-oniguruma.1/opam?